.github/workflows/pr.yml

```
@@ -0,0 +1,454 @@
+        # Determine risk level
+        RISK_LEVEL="LOW"
+        RISK_REASONS=""
+        
+        if [ $TOTAL_FILES -gt 20 ]; then
+          RISK_LEVEL="HIGH"
+          RISK_REASONS="$RISK_REASONS\n- Large number of files changed ($TOTAL_FILES)"
+        elif [ $TOTAL_FILES -gt 10 ]; then
+          RISK_LEVEL="MEDIUM"
+          RISK_REASONS="$RISK_REASONS\n- Moderate number of files changed ($TOTAL_FILES)"
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)